R/sysreqs2.R

```diff
@@ -21,6 +21,7 @@ sysreqs2_cmds <- utils::read.table(
    'Fedora Linux'             linux   fedora       *         NA                  'dnf install -y'                    rpm
    'openSUSE Linux'           linux   opensuse     *         NA                  'zypper --non-interactive install'  rpm
    'SUSE Linux Enterprise'    linux   sle          *         NA                  'zypper --non-interactive install'  rpm
+   'Alpine Linux'             linux   alpine       *         NA                  'apk add --no-cache'                apk
```

Comment:
Why the `--no-cache`?

Reply:
Alpine is often used in a containerized context where one usually does not want to store the cache of a package manager. 
I'd argue that > 90% of it's use will result in an image or executed within an existing image. Omitting the cache part there saves some space, even though of debatable size.

Yes, this does not align with the behavior of the other package managers currently, even though those might more often be used in a non-containerized environment, in which using a cache is helpful. Let me know if I should remove it.

Reply:
Yeah, I am torn between making it behave like other platforms and doing the thing that people usually want. I am leaning towards using the `--no-cache`. 

OTOH you already have to take an extra step to clean pak's cache. So maybe we should add a global `--no-cache` option to pak/pkgdepends itself, and then it would not cache any R packages and it also would not cache system packages. This is kind of available with `*=?nocache`, but I doubt that many people know about that, and it is also hard to set globally.

Btw. are you using https://github.com/r-hub/r-minimal or your own image? The `installr` script on r-minimal already cleans the pak & apk cache manually, so if you are using that, then it does not really matter what we decide here.

Anyway, let's keep it for now.

Reply:
`clean()` deletes everything, no? Anyway, please open an issue if you have a problem this.